test/utils/client/clients.go

```diff
@@ -5,14 +5,17 @@ import (
 
 	"github.com/golang/glog"
 
+	ptpv1 "github.com/openshift/ptp-operator/pkg/client/clientset/versioned/typed/ptp/v1"
```

Comment:
nit: move this one back to the last part please :)

Reply:
done